### ISSUE TITLE
New website is not mobile friendly

### ISSUE THREAD

**Issue by pbb72** [mapillary/mapillary_issues#2122]
While the new websites works on a (powerful) mobile phone, it's not optimised.

Which is a bit weird for a company that is relying so heavily on mobile technology and being out in the field.

When someone is asking what I am doing, it would be nice to be able to show them the website on my phone.

**Comment by gyllen** [comment]
@pbb72 Its gonan take some time to optimize the viewer for mobile. Our plan for now is to

1. Optimize static sites for mobile.
2. Direct you to open an image in app when you hit the viewer.

How does that sound?

**Comment by pbb72** [comment]
I'm not sure what you mean by static sites. The problem with requiring apps is that this scares away people who just want to check out what Mapillary is, or just want to see a one-off image.

**Comment by gyllen** [comment]
@pbb72 By static website I mean all pages that are not the viewer (landing, community and so on). 

I understand your concern. Im just pointing out that it might to take some time for us to fix it (since we have some stuff that we want todo before) and in the meanwhile we have to go for the simplest solution. Designing the viewer for the phone is months of work :)

**Comment by pbb72** [comment]
No problem, like I said the site currently does work in mobile, so this is definitely not high priority. :+1:

**Comment by nicolas17** [comment]
The site doesn't actually work on my iPhone 6. It runs out of memory, crashes the Safari web content process, and reloads. This repeats a few times until Safari gives up and shows an error message.

**Comment by gyllen** [comment]
@nicolas17 Yeah what we want todo is if possible just redirect you to the photo in the app. See my comments above about making it mobile friendly. Its not the highest prio right now.

**Comment by HolgerJeromin** [comment]
IMO the main page is not usable on mobile (tested mobile opera) at all. 
Double click into the map zooms the page and i cannot reach the controls because of the map panning.

But i was really surprised that the android app has a quite nice and really usable minimal version of the new website. 
IMO this version should be used on mobile right now.

**Comment by gyllen** [comment]
@HolgerJeromin Spot on! I think thats the path we want to take. Show a degraded experience in the phones browser but add a link to open in app instead.

**Comment by HolgerJeromin** [comment]
I do not see why you need/want the app for this small version.

**Comment by gyllen** [comment]
I think we are misunderstanding each other. I  mean that we should steer you towards opening the image in the App, when you are on the phone.

**Comment by knikel** [comment]
Assigning @amahon to consider this as a feature (Responsive mapiilary.com).

**Comment by Kaldea** [comment]
_mapillary_issues_ is no longer an active repository.

Let’s continue discussing features in our forum! We will import this issue as topic to the [Mapillary Forum](http://forum.mapillary.com) in the coming weeks. We’ll notify here once it’s been done. See you there!
